packages/repository/README.md

```diff
@@ -291,15 +311,14 @@ import {
   Entity,
   Options,
   DataObject,
-  Filter,
   EntityCrudRepository,
 } from '@loopback/repository';
 
 // The Controller for Note
 class NoteController {
   constructor(
     // Use constructor dependency injection to set up the repository
-    @repository('noteRepo')
+    @repository('repositories.noteRepo')
```

Comment:
It's my understanding that the `@repository` sugar methods perform this step for you. The new example would have an injection key of `repositories.repositories.noteRepo` if done this way.

Reply:
That is correct. Good catch!